File: calc_media.py
def calcular_media(numeros):
    soma = 0
    for numero in numeros:
        soma += numero
    return soma / len(numeros)

def main():
    numeros = [10, 20, 30, 40, 50]
    media = calcular_media(numeros)
    print(f"A média é: {media:.2f}")

if __name__ == "__main__":
    main()

File: calc_media.c
#include <stdio.h>

float calcular_media(int numeros[], int tamanho) {

    int soma = 0;

    for(int i = 0; i < tamanho; i++) {

        soma += numeros[i];

    }

    return soma / (float)tamanho;

}

int main() {

    int numeros[] = {10, 20, 30, 40, 50};

    int tamanho = sizeof(numeros) / sizeof(numeros[0]);

    float media = calcular_media(numeros, tamanho);

    printf("A média é: %.2f\n", media);

    return 0;

}
